app/test_main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+from unittest import mock
+import pytest
+
+from app.main import can_access_google_page
+
+
+class TestCanAccessGoogle:
```

Comment:
There is no need to create `TestCanAccessGoogle` class, you can place the test function aside from it and remove the class